server/src/main/java/org/bonitasoft/web/rest/server/api/bpm/process/ProcessInstantiationResource.java

```diff
@@ -63,11 +66,16 @@ public String instantiateProcess(final Map<String, Serializable> inputs) throws
             final long tenantId = apiSession.getTenantId();
             final long maxSizeForTenant = PropertiesFactory.getConsoleProperties(tenantId).getMaxSize();
             final Map<String, Serializable> processedInputs = typeConverterUtil.getProcessedInput(processContract, inputs, maxSizeForTenant, tenantId);
+            long processInstanceId;
```

Comment:
we ought to use caseId instead of processInstanceId since it doesn't mean antyhing on the web side, shouldn't we ?